### PR TITLE
Andy collections submission

### PR DESCRIPTION
**Pull request by amitchell94**

### CODE REVIEW

File: src/javafundamentals/collections/list/Ex03MaxValue.java [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
  * Create a List with 20 random numbers.
  * Find the maximum value of the list and print it.
  */
 
 public class Ex03MaxValue {
+    public static void main(String[] args) {
+        List<Integer> randomNoList = new ArrayList<>();
+
+        for (int i = 0; i < 20; i++) {
+            randomNoList.add((int)(Math.random() * 100 -1) +1);
```
Comment (roioteromorales) [inline src/javafundamentals/collections/list/Ex03MaxValue.java:17]: You can use the class Random instead:
```
Random randomGenerator = new Random();
int randomNumber = randomGenerator.nextInt();
```

File: src/javafundamentals/collections/list/Ex04DeleteElements.java [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 
 public class Ex04DeleteElements {
+    public static void main(String[] args) {
+        List<Integer> numberList = new ArrayList<>();
+
+        for (int i = 1; i <= 10; i++) {
+            numberList.add(i);
+        }
+
+        numberList.remove(numberList.indexOf(7));
```
Comment (roioteromorales) [inline src/javafundamentals/collections/list/Ex04DeleteElements.java:20]: there is another method that removes the Object that is passed on.
So if you create an Integer object you can pass it to remove the object without having to use the `indexOf()` method
`            numberList.remove(new Integer(7));
`

File: src/javafundamentals/interfaces/Englishman.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package javafundamentals.interfaces;
+
+public class Englishman implements Person {
+
+
+    static void makeTea(){
```
Comment (roioteromorales) [inline src/javafundamentals/interfaces/Englishman.java:6]: this method doesnt need to be static

File: src/javafundamentals/collections/list/Ex02Iterate.java [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
 public class Ex02Iterate {
+    public static void main(String[] args) {
+        List<Integer> integerList = new ArrayList<>();
+
+        for (int i = 0; i <= 10; i++) {
+            integerList.add(i);
+        }
+
+        int total = 0;
+        for (int i:
```
Comment (roioteromorales) [inline src/javafundamentals/collections/list/Ex02Iterate.java:20]: Just small detail about formatting, i notice that you always format the foreach loops as this one, the standard way is:

```
for (int i: integerList) {
    total += i;
}
```

**Review by roioteromorales — APPROVED** [review]
All good! i left some comments for you to check as improvements!

File: src/javafundamentals/collections/list/Ex03MaxValue.java (hunk 1 above)
Comment (amitchell94) [reply src/javafundamentals/collections/list/Ex03MaxValue.java:17]: Okay, I will try to remember to do that next time. Thanks

File: src/javafundamentals/collections/list/Ex04DeleteElements.java (hunk 2 above)
Comment (amitchell94) [reply src/javafundamentals/collections/list/Ex04DeleteElements.java:20]: Right I see, thanks!

File: src/javafundamentals/interfaces/Englishman.java (hunk 3 above)
Comment (amitchell94) [reply src/javafundamentals/interfaces/Englishman.java:6]: Okay thanks, good point!

File: src/javafundamentals/collections/list/Ex02Iterate.java (hunk 4 above)
Comment (amitchell94) [reply src/javafundamentals/collections/list/Ex02Iterate.java:20]: Ah okay, intelliJ always puts the bracket on the second line down so I thought i would copy that. Good to know!

**Pull request merged**